modules/jupyter-widget/src/version.js

```diff
@@ -4,11 +4,5 @@
  * The html widget manager assumes that this is the same as the npm package
  * version number.
  */
-/* global __VERSION__ */
-export const MODULE_VERSION =
-  typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'untranspiled source';
-
-/*
- * The current package name.
- */
+export const MODULE_VERSION = '7.4.0-alpha.2';
```

Comment:
Are we supposed to manually update this every time we publish?

Reply:
In the [widget cookiecutter's version file](https://github.com/jupyter-widgets/widget-ts-cookiecutter/blob/master/%7B%7Bcookiecutter.github_project_name%7D%7D/src/version.ts) they read the module version from the `package.json`, which we could in theory do as well.